.github/workflows/plugin-delivery.yml

```diff
@@ -77,12 +77,32 @@ jobs:
           module_name: plugins
           distrib: ${{ matrix.distrib }}
           cache_key: ${{ github.sha }}-${{ github.run_id }}-rpm-${{ matrix.distrib }}
+          stability: ${{ inputs.stability }}
+          artifactory_token: ${{ secrets.artifactory_token }}
+
+  deliver-rpm-legacy:
+    runs-on: [self-hosted, common]
+    strategy:
+      matrix:
+        distrib: [el7, el8]
+        major_version: ["21.10", "22.04", "22.10"]
```

Comment:
We can remove the delivery to 21.10 not supported any more

Reply:
seems still maintained until 11/2023 : https://docs.centreon.com/fr/docs/releases/lifecycle/#tableau-de-maintenance-des-versions